packages/@react-spectrum/list/src/listview.css

```diff
@@ -56,27 +85,27 @@
       border-color: var(--spectrum-global-color-blue-500);
       &.is-hovered,
       &.is-active {
-        background-color: var(--spectrum-table-row-background-color-selected);
+        background-color: var(--spectrum-table-row-background-color-selected-hover);
       }
 
       /* avoid double border for adjacent selected items */
       &.is-previous-selected {
-        &:not(.is-focused) {
-          border-top-color: transparent;
-        }
+        border-top: none;
```

Comment:
Causes the height to jump, not sure if there is a better solution or not:

https://user-images.githubusercontent.com/8961049/154162572-477ba6fc-10a5-4932-8981-d6ae703d02ed.mov